modules/Hoot/ui/managePanel/basemaps.js

```diff
@@ -52,18 +53,38 @@ export default class Basemaps extends Tab {
             let basemaps = await Hoot.api.getBasemaps();
 
             this.populateBasemaps( basemaps );
+
         } catch ( e ) {
             window.console.error( 'Unable to retrieve basemaps' );
             throw new Error( e );
         }
     }
 
+    checkBasemapStatus(basemaps) {
+        let basemap = {
+            available: [],
+            unavailable: []
+        };
+        _forEach(basemaps, function (d) {
+
+            if (d.status === 'enabled' || d.status === 'disabled') {
+                basemap.available.push(d);
+            }
+            else {
+                basemap.unavailable.push(d);
+            }
+        });
+        return basemap;
+    }
+
     populateBasemaps( basemaps ) {
         let instance = this;
 
+        let allBasemaps = this.checkBasemapStatus( basemaps );
```

Comment:
Related to the GIF i posted, maybe here it just is a filter so that only those that match `d.status==='enabled' || d.status==='disabled'` are returned